spec/v2/index.bs

```diff
@@ -2072,7 +2074,16 @@ The [=PF Request Event=] is defined as a JSON-encoded CloudEvent event with the
 with
 
 : <dfn>ProductFootprintFragment</dfn>
-:: A JSON object which references a subset of <{ProductFootprint}> properties, **including nested properties**.
+:: A JSON object which references a subset of <{ProductFootprint}> properties, **including nested properties**. It is RECOMMENDED to include at least the [=ProductIds=] in the PF Request Event request body.
```

Comment:
Shouldn't we be more strict here? From my point of view productID should be a MUST. Otherwise the provider side cannot identify the PCF it should calculate and/or send. 

Can we create a list of fileds we see as mandatory to be supported by the PCF provider? Shouldn't be that much... ;)

Reply:
Thank you!
I just want to point out here that this suggestion would be a backwards-breaking change and that the ADR would have to be updated accordingly.